src/App.tsx

```diff
@@ -1,61 +1,120 @@
+import React, { FC, useState } from 'react';
+
+import { TodoList } from './components/TodoList';
+import { User } from './Types/User';
+import { Todo } from './Types/Todo';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUser(userId: number): User | null {
```

Comment:
```suggestion
function getUserById(userId: number): User | null {
```

Reply:
it is better to specify our function - be get user by id here, so name it correctly `getUserById`